src/Day.jsx

```diff
@@ -240,17 +301,7 @@ export default class Day extends PureComponent {
             frozen
           />
         ))}
-        <div
-          onMouseDown={this.handleMouseDown}
-          onMouseUp={this.handleMouseUp}
-          onMouseMove={this.handleMouseMove}
-          onMouseOut={this.handleMouseUp}
-          onTouchStart={this.handleTouchStart}
-          onTouchMove={this.handleTouchMove}
-          onTouchEnd={this.handleTouchEnd}
-          className={styles.mouseTarget}
-          ref={this.handleMouseTargetRef}
-        />
+        <this.EventTracking available={available} hourLimits={hourLimits} />
```

Comment:
Instead of initializing the component on the instance, did you consider something like 

```jsx
{available && (
<div
  onMouseDown={this.handleMouseDown}
  onMouseUp={this.handleMouseUp}
  onMouseMove={this.handleMouseMove}
  onMouseOut={this.handleMouseUp}
  onTouchStart={this.handleTouchStart}
  onTouchMove={this.handleTouchMove}
  onTouchEnd={this.handleTouchEnd}
  className={styles.mouseTarget}
  ref={this.handleMouseTargetRef}
  style={{ 
    top: hourLimits.top,
    height: hourLimits.difference,
  }}
/>
)
```

While I appreciate the effort to reduce the size of the render method, I think having things like these inline is more straightforward.

Reply:
Good idea, I'll make the necessary changes.